tools/install/Extra/additional-files-to-sign.txt

```diff
@@ -6,15 +6,6 @@
 Greg.dll
 Analytics.Net.ADP.dll
 Analytics.NET.Core.dll
-LibG.Interface.dll
 ProtoGeometry.customization.dll
-ProtoGeometry.dll
```

Comment:
what are these changes about?

Reply:
Yes, they are already signed as part of the LibG build.